README.md

```diff
@@ -66,15 +66,15 @@ Design proposals in this repository are expected to be well thought-through and
 a prototype implementation that demonstrates their feasibility. All design proposals in this repository
 shall be backed by maintainers of the corresponding subsystems in the Kotlin compiler or its standard library.
 
-If you are in doubt of whether the proposal meets those criteria, please start with the discussion
-of your idea, use-case, or a specific enhancement in the appropriate channels and secure support for your general
+If you are in doubt about whether the proposal meets those criteria, please start with the discussion
+of your idea, use case, or a specific enhancement in the appropriate channels and secure support for your general
 idea/proposal from maintainers before submitting a KEEP.
 
 We will be gradually moving KEEPs that are not backed by Kotlin maintainers into YouTrack issues for further 
 discussions. 
 
 ### Contributing to existing KEEPs
 
-* For in-progress KEEPs, please keep discussions in the corresponding issue.
+* For in-progress KEEPs, please keep discussions on the corresponding issue.
```

Comment:
The intent of this phrase was to direct discussion "_in_ the corresponding GitHub issue", as opposed to discussing it elsewhere (in PR, in forums, emails, slack, etc).

Reply:
done